Sources/UInt128.swift

```diff
@@ -373,204 +325,168 @@ extension UInt128: Strideable {
     /// data type does not exist, so it has to fall back to an IntMax
     /// representation which lacks half of the storage space when end
     /// is less than the value of self.
-    public func distanceTo(_ end: UInt128) -> Stride {
```

Comment:
I agree. Add what the 3.1 API has, leave the cruft, then in a subsequent release remove the cruft.